pkg/playground/speed.js

```diff
@@ -7,6 +7,7 @@ var timer = null;
 var start = null;
 var total = 0;
 var proc = null;
+var close_problem;
```

Comment:
`var`? :)

Reply:
Let's keep the current file's code style. When I (or anyone else) is bored, we can convert it wholesale to some nice ES6 :grin: